library/include/rocblas-functions.h

```diff
@@ -8838,6 +8838,664 @@ ROCBLAS_EXPORT rocblas_status rocblas_zsyr2_strided_batched(rocblas_handle
 
     \details
 
+    herk performs one of the matrix-matrix operations for a Hermitian rank-k update
+
+    C := alpha*op( A )*op( A )^H + beta*C
+
+    where  alpha and beta are scalars, op(A) is an n by k matrix, and
+    C is a n x n Hermitian matrix stored as either upper or lower.
+
+        op( A ) = A   or   op( A ) = A^H
+
```

Comment:
can you move the information defining op(A) from the transA comment to here 
op( A ) = A, and A is  n by k  if transA  == rocblas_operation_none      
op( A ) = A^H and A is k by n  if transA  == rocblas_operation_conjugate_transpose